src/xFactor/Judge.java

```diff
@@ -0,0 +1,104 @@
+package xFactor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Judge extends Person {

```

Comment:
none of its fields should be static. Static fields are class variables, a Judge should live independently of other Judge instances